src/processTargets/modifiers/scopeTypeStages/ItemStage.ts

```diff
@@ -0,0 +1,361 @@
+import { Range, TextEditor } from "vscode";
```

Comment:
Please move into a directory so that you can factor out all the different functions